paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/source/FileStoreSourceSplitReader.java

```diff
@@ -62,14 +64,20 @@ public class FileStoreSourceSplitReader
     private RecordIterator<InternalRow> currentFirstBatch;
 
     private boolean paused;
+    private final FileStoreSourceReaderMetrics sourceReaderMetrics;
+    private long currentSnapshotId = FileStoreSourceReaderMetrics.UNDEFINED;
 
-    public FileStoreSourceSplitReader(TableRead tableRead, @Nullable RecordLimiter limiter) {
+    public FileStoreSourceSplitReader(
+            TableRead tableRead,
+            @Nullable RecordLimiter limiter,
+            FileStoreSourceReaderMetrics sourceReaderMetrics) {
```

Comment:
`sourceReaderMetrics` is set to `null` in `FileStoreSourceSplitReaderTest`. Add `@Nullable` here.